libsolidity/interface/CompilerStack.h

```diff
@@ -145,6 +145,13 @@ class CompilerStack: boost::noncopyable
 	/// Must be set before parsing.
 	void setEVMVersion(langutil::EVMVersion _version = langutil::EVMVersion{});
 
+	/// Sets the list of requested sources.
```

Comment:
This looks weird to me at first, because why would you hand over a source file to the compiler if you do not want to compile it after all, but it might be useful still.